### PR TITLE
Find Similar Incidents  - bug

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29185

## Description
fix the encode / decode bug
## Screenshots
Create a function that illustrates the error in the customer -
image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ChanochShayner** [comment]
The build failed with the same customer error -

**Comment by ChanochShayner** [comment]
@adi88d 
Please note the changes, please check that it does not destroy what you have fixed in this function.

File: Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py [hunk 1]
```diff
@@ -118,6 +118,65 @@ def execute_command(command, args=None):
         return []
 
 
+def test_get_incidents_by_keys():
+    from FindSimilarIncidentsV2 import get_incidents_by_keys
+    res = get_incidents_by_keys({u'name': u'\U0001f489'}, 'created', '2020-10-07T19:49:37.392378+03:00', '7137', 72,
```
Comment (teizenman) [inline Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py:123]: Change the name to a real emoji, not the encoded one.
Like here: https://github.com/demisto/content/blob/master/Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2_test.py#L81

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py (hunk 1 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py:123]: If there is an emoji in the incident **this** function (`get_incidents_by_keys`) accepts it that way - u'\U0001f489'

**Review by teizenman — APPROVED** [review]

File: Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py [hunk 2]
```diff
@@ -114,11 +114,10 @@ def get_incidents_by_keys(similar_incident_keys, time_field, incident_time, inci
 
     similar_keys_list = []
     for key, value in similar_incident_keys.items():
+        value = value.encode('utf-8')
         compare_str = '{}:={}' if str(value).isdigit() else '{}="{}"'
-        similar_keys_list.append(compare_str.format(key, str(value)
-                                 .replace('"', r'\"')
-                                 .replace("\n", "\\n").
-                                 replace("\r", "\\r")))
+        similar_key = compare_str.format(key, str(value).replace('"', r'\"').replace("\n", "\\n").replace("\r", "\\r"))
```
Comment (guykeller) [inline Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py:119]: This isn't very readable at the moment in my opinion.
However, leaving this to your judgement @teizenman @ChanochShayner

File: Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py [hunk 3]
```diff
@@ -118,6 +118,65 @@ def execute_command(command, args=None):
...
+         'notifyTime': '2019-02-20T15:47:27.156966+02:00',
+         'occurred': '2019-02-20T15:47:23.962163+02:00',
+         'openDuration': 0,
+         'owner': 'analyst',
+         'parent': '',
+         'phase': '',
+         'playbookId': 'playbook0',
+         'previousRoles': None,
+         'rawCategory': '',
+         'rawCloseReason': '',
```
Comment (guykeller) [inline Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py:161]: This should definitely be extracted into a variable with a readable name, this test is very hard to read and understand at the moment

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py (hunk 3 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py]: Done

**Review by guykeller — APPROVED** [review]

**Pull request merged**